### DISCUSSION TITLE
tutorials/voyage-part1/05-managed-federation-and-the-supergraph

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#168] [course feedback]
# tutorials/voyage-part1/05-managed-federation-and-the-supergraph

Learn about the pieces of a federated architecture and how they fit together

https://www.apollographql.com/tutorials/voyage-part1/05-managed-federation-and-the-supergraph

**Comment by pgn666** [comment]
IMGO, in the practice exercise it should be:
"With [Rover CLI] each of these schemas is published to the Apollo schema registry"

**Reply by rcooper47** [reply]
Agreed.

**Comment by shauryam** [comment]
Instead of polling the Uplink, why a webhook hasn't been used? After all the router is a server which should be able to handle a webhook
